gui/wxpython/nviz/wxnviz.py

```diff
@@ -2388,36 +2397,36 @@ def Load(self):
 
         return Nviz_load_image(im, self.width, self.height, self.image.HasAlpha())
 
-    def Draw(self):
+    def Draw(self) -> None:
         """Draw texture as an image"""
         Nviz_draw_image(
             self.coords[0], self.coords[1], self.width, self.height, self.textureId
         )
 
-    def HitTest(self, x, y, radius):
+    def HitTest(self, x: int, y: int, radius) -> bool:
```

Comment:
radius has no annotation

Reply:
```suggestion
    def HitTest(self, x: int, y: int, radius: int) -> bool:
```
I know why I didn't find it at the time, I just had to go dig a little deeper in the wxPython docs to find my answer